qlib/utils/__init__.py

```diff
@@ -728,3 +728,23 @@ def load_dataset(path_or_obj):
     elif extension == ".csv":
         return pd.read_csv(path_or_obj, parse_dates=True, index_col=[0, 1])
     raise ValueError(f"unsupported file type `{extension}`")
+
+
+#################### Operator Wrapper #####################
+
+
+class OpsWrapper(object):
+    """Ops Wrapper"""
+
+    def __init__(self):
+        self._ops = {}
+
+    def register(self, ops_list):
+
+        for operator in ops_list:
+            self._ops[operator.__name__] = operator
+
+    def __getattr__(self, key):
+        if self._ops is {}:
+            raise AttributeError("Please run qlib.init() first using qlib to register ops")
```

Comment:
The Ops is not register in qlib.init.
The error info is not aligned with the code.